README.md

```diff
@@ -44,14 +43,23 @@ Initialize the SDK in your `main` function. This starts the SDK with the default
 import 'package:instabug_flutter/instabug_flutter.dart';
 
 void main() {
-  WidgetsFlutterBinding.ensureInitialized();
-
-  Instabug.init(
-    token: 'APP_TOKEN',
-    invocationEvents: [InvocationEvent.shake],
+  runZonedGuarded(
+    () {
+      WidgetsFlutterBinding.ensureInitialized();
+
+      Instabug.init(
+        token: 'APP_TOKEN',
+        invocationEvents: [InvocationEvent.shake],
+      );
+
+      FlutterError.onError = (FlutterErrorDetails details) {
+        Zone.current.handleUncaughtError(details.exception, details.stack!);
+      };
+
+      runApp(MyApp());
+    },
+    CrashReporting.reportCrash,
```

Comment:
@asamiz I believe this change should be in the code block under the "Crash Reporting" section not "Initializing Instabug" since we don't actually use `runZonedGuarded` here, we use it with crash reporting as you can see in the code block under its section.

Reply:
@a7medev  Okay makes sense, but what should I add for the initialization code?

Reply:
@asamiz I think the initialization code (under "Initializing Instabug") doesn't need changes since it already doesn't use `runZonedGuarded` which is the cause of the issue. So we only need this in the crash reporting section (let's also keep the initialization code there the same way we will do with the documentation).

Reply:
Okay, thank you @a7medev!
Done.